docs/src/snap/explanation/datastore/external.md

```diff
@@ -0,0 +1,49 @@
+# external datastore
```

Comment:
Here and elsewhere: Is there a guideline/convention to start with capital letters for a headline? @evilnick